fapolicy_analyzer/ui/strings.py

```diff
@@ -6,6 +6,7 @@
 SYSTEM_INITIALIZATION_ERROR = _("Error initializing System")
 ANCILLARY_TRUST_LOAD_ERROR = _("Error loading Ancillary Trust")
 SYSTEM_TRUST_LOAD_ERROR = _("Error loading System Trust")
+DAEMON_INITIALIZATION_ERROR = _("Error initializing communications with daemon")
```

Comment:
You need to run `pipenv run extract_messages` to update the pot file.

Reply:
Done